.circleci/config.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    machine: true
```

Comment:
@ozbillwang Sorry for the delay, but I'm not sure I'm follow.  Check out this config, which I think does what you're trying to do, although it does do the TF linting your trying to do with your Makefile 

```Yaml
version: 2

jobs:
  build:
    docker:
      - image: fitnesskeeper/terraform:v0.9.11
        entrypoint: /bin/sh
    steps:
      - checkout
      - run:
          name: terraform-validate
          command: terraform validate

workflows:
  version: 2
  build:
    jobs:
      - build
```